From 127e40dcc2d62ad693cb4e03e1182d0a227d85e1 Mon Sep 17 00:00:00 2001
From: Evangelos Foutras <evangelos@foutrelis.com>
Date: Sat, 16 Apr 2022 08:24:58 +0300
Subject: [PATCH] Fix uuid/uuid.h detection with util-linux 2.38

util-linux commit 10f5f79485964ab52272ebe79c3b0047b1f84d82 changed the
header guard in uuid/uuid.h from _UUID_UUID_H to _UL_LIBUUID_UUID_H as
the former was too generic; fix guid.cc to also look for the new name.
---
 guid.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/guid.cc b/guid.cc
index 1e73ab7..d3e4fd5 100644
--- a/guid.cc
+++ b/guid.cc
@@ -141,7 +141,7 @@ void GUIDData::Zero(void) {
 void GUIDData::Randomize(void) {
    int i, uuidGenerated = 0;
 
-#ifdef _UUID_UUID_H
+#if defined (_UUID_UUID_H) || defined (_UL_LIBUUID_UUID_H)
    uuid_generate(uuidData);
    ReverseBytes(&uuidData[0], 4);
    ReverseBytes(&uuidData[4], 2);
